PF/Assignment3/Binary_Search.java

```diff
@@ -0,0 +1,51 @@
+import java.util.*;
+public class Binary_Search 
```

Comment:
ACTION: Don't use '_' in class name. Please check following link for proper naming conventions.
[Coding convetion guideline](http://www.javatpoint.com/java-naming-conventions)